src/Button/types.ts

```diff
@@ -34,7 +34,7 @@ export type ButtonBaseProps = {
    */
   disabled?: boolean
 } & SxProp &
-  HTMLAttributes<HTMLButtonElement> &
+  React.ButtonHTMLAttributes<HTMLButtonElement> &
```

Comment:
we actually only want button attributes, not all possible attributes (I think)

Reply:
happy to revert this (and the similar) changes if that makes sense.  I realize this is out of scope in most places, potentially, although also maybe more accurate